core-bundle/tests/Contao/TemplateTest.php

```diff
@@ -226,4 +227,20 @@ public function testLoadsTheAssetsPackages(): void
         $template = new FrontendTemplate();
         $template->asset('/path/to/asset', 'package_name');
     }
+
+    public function testCanDumpTemplateVars(): void
+    {
+        $template = new FrontendTemplate();
```

Comment:
```suggestion
        /** @var FrontendTemplate&MockObject $user */
        $template = $this->mockClassWithProperties(FrontendTemplate::class);
```

Please test if this fixes the tests on Travis.